public/styles/helpers/_icons.scss

```
@@ -68,3 +68,47 @@
     vertical-align: middle;
   }
 }
+
+.header-discord {
+  svg {
+    circle {
+      transition: all 0.3s;
+      fill: $jumbo-color;
+    }
+
+    path {
+      fill: #2f3241;
+    }
+  
+    &.active,
+    &:hover{
+      circle {
+        fill: $jumbo-color-strong;
+      }
+    }
+  
+    &:active {
+      circle {
+        fill: $jumbo-color-subtle;
+      }
+    }
+  }
+}
+
+.footer-discord {
+  svg {
+    circle {
+      color: $main-color-strong;
```

```suggestion
      fill: $main-color-strong;
```

Renders as black using the `color` prop.